File: C++ solutions/706. Design HashMap.cpp
class MyHashMap {
public:
    vector<int>HashMap;
    MyHashMap() {
        HashMap.resize(1000007, -1);
    }
    
    void put(int key, int value) {
        HashMap[key] = value;
    }
    
    int get(int key) {
        return HashMap[key];
    }
    
    void remove(int key) {
        HashMap[key] = -1;
    }
};

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap* obj = new MyHashMap();
 * obj->put(key,value);
 * int param_2 = obj->get(key);
 * obj->remove(key);
 */

File: Python solutions/706. Design HashMap.py
class MyHashMap:

    def __init__(self):
        self.HashMap = [-1] * 1000007        

    def put(self, key: int, value: int) -> None:
        self.HashMap[key] = value

    def get(self, key: int) -> int:
        return self.HashMap[key]

    def remove(self, key: int) -> None:
        self.HashMap[key] = -1


# Your MyHashMap object will be instantiated and called as such:
# obj = MyHashMap()
# obj.put(key,value)
# param_2 = obj.get(key)
# obj.remove(key)